setup.py

```diff
@@ -72,13 +72,14 @@ def build_extension(self, ext):
     raise RuntimeError('Unable to determine version.')
 
 
-install_requires = ['attrs>=17.3.0', 'chardet>=2.0,<4.0',
-                    'multidict>=4.0,<5.0',
-                    'async_timeout>=3.0,<4.0',
-                    'yarl>=1.0,<2.0']
-
-if sys.version_info < (3, 7):
-    install_requires.append('idna-ssl>=1.0')
+install_requires = [
+    'attrs>=17.3.0',
+    'chardet>=2.0,<4.0',
+    'multidict>=4.0,<5.0',
+    'async_timeout>=3.0,<4.0',
+    'yarl>=1.0,<2.0',
+    'idna-ssl>=1.0;python_version<"3.7"',
```

Comment:
AFAIR pip (setuptools?) doesn't support this kind of notation in `install_requires` (at least in certain range of versions).
I'd say it's better to go for extras-based syntax:
```python
extra_requires = {
    ':python_version<"3.7"': [
        'idna-ssl>=1.0',
    ],
}
```

Reply:
Is there any way to test this? There have been quite a few improvements in this area in pip v10.x, and support for 508 came up more than once.

Reply:
@webknjaz it does today. And at least bundled pip for 3.5 support that. For older I think we shouldn't care. 

In anyway, rule of thumb is: before install anything in new virtualenv, upgrade pip and setuptools first.